### PR TITLE
File Added : All changed files

### PR DESCRIPTION
**Pull request by yogesh545**
Indentation applied

### CODE REVIEW

**Review by Richa199 — APPROVED** [review]

**Review by Jestinxavier — APPROVED** [review]

**Review by mubeen2710 — APPROVED** [review]

File: yogesh_khande/html/asgn1/Intro1.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-<head>
-<title>assignment1</title>
-</head>
-<body>
-<h1><b>HTML</b></h1>
-<h4><b>Introduction</b></h4>
-<p>HTML is a standard markup language for web page.</p>
-</body>
+  <head>
+    <title>Question1</title>
```
Comment (KomalShinde813) [inline yogesh_khande/html/asgn1/Intro1.html]: change title, title should be related to program

File: yogesh_khande/html/asgn1/Intro2.html [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-<h4><b>Introduction</b></h4>
-<p> HTML is a standard markup language for web page.</p>
-<h3>HTML Editor</h3>
-<h4><b>Notepad or TextEdit.</b></h4>
-<p>web pages can be created and modified y using professional HTML editors.
-However, for learing HTML we recommend a simple text editor like NotePad(PC) or TextEdit(MAC).
-</p>
-</body>
+  <head>
+    <title>Question2</title>
```
Comment (KomalShinde813) [inline yogesh_khande/html/asgn1/Intro2.html]: change title, title should be related to program

File: yogesh_khande/html/asgn1/Intro3.html [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
-<p> HTML is a standard markup language for web page.</p>
-<hr>
-<h3>HTML Editor</h3>
-<h4><b>Notepad or TextEdit.</b></h4>
-<p>web pages can be created and modified by using professioonal HTML editors.
-However, for learning HTML we recommend a simple text editor like Notepad(PC) or TextEdit(MAC).
-</p>
-</body>
+  <head>
+    <title>assignment3</title>
```
Comment (KomalShinde813) [inline yogesh_khande/html/asgn1/Intro3.html]: change title, title should be related to program

File: yogesh_khande/html/asgn1/Myinfo.html [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
-<body>
-<h1><b>GitHub User</b></h1>
-<h2><b>Introduction</b></h2>
-<hr>
-<p> I am yogesh khande a new github user with id: yogesh545
-I completed my BE degree in Mechanical at PCE college Nagpur
-</p>
-</body>
+  <head>
+    <title>assignment4</title>
```
Comment (KomalShinde813) [inline yogesh_khande/html/asgn1/Myinfo.html]: change title, title should be related to program

File: yogesh_khande/html/assignment3/Home.html [hunk 5]
```diff
@@ -1,39 +1,39 @@
...
+  </head>
+  <body bgcolor="grey">
+       <h1 style="color: rgb(78, 71, 175);"><center>Ambersoft</center></h1>
+      <center>
+       <a href="#HOME">HOME</a>  
+       <a href="#MI-11-MEAN">|MI-11-MEAN|</a>
+       <a href="#CONTACT">CONTACT</a>
+      </center>  
+      <center><H1 id="HOME">HOME PAGE</H1></center>
+      <div class="contaner" style="display:flex;">
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: space required after : in style

File: yogesh_khande/html/assignment3/Home.html [hunk 6]
```diff
@@ -1,39 +1,39 @@
...
+      <img src="https://t4.ftcdn.net/jpg/02/71/22/77/360_F_271227781_gt3nHI7H3CgUENSbM3V14x2QhPB04Glb.jpg"  style="float: left" width="70%" margin:50px padding-left:50px>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: space required after : in style

Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: remove extra space which is present before style

File: yogesh_khande/html/assignment3/Home.html [hunk 7]
```diff
@@ -1,39 +1,39 @@
...
        this is home page this is home page this is home page this is home page this is home page this is home page this is home page
        this is home page this is home page this is home page this is home page this is home page this is home page this is home page
        this is home page this is home page
-    </p>
-    <img src="https://t4.ftcdn.net/jpg/02/71/22/77/360_F_271227781_gt3nHI7H3CgUENSbM3V14x2QhPB04Glb.jpg"
-    style="float: right;"width=70% margin:50px>
-    </div>
+      </p>
+      <img src="https://t4.ftcdn.net/jpg/02/71/22/77/360_F_271227781_gt3nHI7H3CgUENSbM3V14x2QhPB04Glb.jpg"
+      style="float: right;"width=70% margin:50px>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: space required after : and each attribute in style

File: yogesh_khande/html/assignment3/Home.html [hunk 8]
```diff
@@ -44,69 +44,68 @@ <h1 style="color: rgb(78, 71, 175);"><center>Ambersoft</center></h1>
...
         <a href="#CONTACT">CONTACT</a>
-    </center>
-     <br>
+       </center>
     <br>
     <br>
     <br>
-  <h1>MI-11-MEAN</h1>
     <br>
+        <h1>MI-11-MEAN</h1>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: indentation is not proper

File: yogesh_khande/html/assignment3/Home.html [hunk 9]
```diff
@@ -44,69 +44,68 @@ <h1 style="color: rgb(78, 71, 175);"><center>Ambersoft</center></h1>
...
     <br>
     <br>
-    <strong>
-    <ul>
-     <li>Snehal</li>
-     <li>Pawan</li>
-     <li>Shankar</li>
-    </ul>
     <br>
+        <strong>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html:72]: check indentation properly

File: yogesh_khande/html/assignment3/Home.html [hunk 10]
```diff
@@ -44,69 +44,68 @@ <h1 style="color: rgb(78, 71, 175);"><center>Ambersoft</center></h1>
...
-    </center>
+      </center>
     <br>
     <br>
     <br>
-    <img src="https://t4.ftcdn.net/jpg/02/71/22/77/360_F_271227781_gt3nHI7H3CgUENSbM3V14x2QhPB04Glb.jpg"
-     style="float: left;" width="25%" margin:50px>
-     <p> contact us contact us contact us contact us contact us contact us contact us contact us contact us contact us
+       <img src="https://t4.ftcdn.net/jpg/02/71/22/77/360_F_271227781_gt3nHI7H3CgUENSbM3V14x2QhPB04Glb.jpg"
+        style="float: left;" width="25%" margin:50px>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: space required after : in style

File: yogesh_khande/html/assignment3/Home.html [hunk 11]
```diff
@@ -44,69 +44,68 @@ <h1 style="color: rgb(78, 71, 175);"><center>Ambersoft</center></h1>
...
+       <p> contact us contact us contact us contact us contact us contact us contact us contact us contact us contact us
         contact us contact us contact us contact us contact us contact us contact us contact us contact us contact us
         contact us contact us contact us contact us contact us contact us contact us contact us contact us contact us
-    </p>
-
-</body>
+       </p>
+  </body>
 </html>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: extra line required at the end of file

File: yogesh_khande/html/assignment4/Table1.html [hunk 12]
```diff
@@ -1,51 +1,50 @@
...
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Question1</title>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment4/Table1.html]: change title, title should be related to program

File: yogesh_khande/html/assignment4/Table2.html [hunk 13]
```diff
@@ -1,68 +1,71 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>table2</title>
-        <style>
-            table,
+  <head>
+    <title>table2</title>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment4/Table2.html]: change title

File: yogesh_khande/html/assignment4/Table2.html [hunk 14]
```diff
@@ -1,68 +1,71 @@
...
-    </body>
-</html>
\ No newline at end of file
+    </style>
+  </head>
+  <body>
+    <table>
+      <caption>
+        <strong style="color: rgb(28, 153, 153); font-size: 30px"
+          >Jan's spring term afternoon schedule</strong
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment4/Table2.html]: closing tag should be at above line and </strong> at below this line

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: yogesh_khande/html/assignment3/Home.html (hunk 6 above)
Comment (KomalShinde813) [reply yogesh_khande/html/assignment3/Home.html]: need to this change  space required after : in style

File: yogesh_khande/html/assignment3/Home.html [hunk 15]
```diff
@@ -1,39 +1,41 @@
...
        this is home page this is home page
-    </p>
-    <img src="https://t4.ftcdn.net/jpg/02/71/22/77/360_F_271227781_gt3nHI7H3CgUENSbM3V14x2QhPB04Glb.jpg"
-    style="float: right;"width=70% margin:50px>
-    </div>
+      </p>
+      <img src="https://t4.ftcdn.net/jpg/02/71/22/77/360_F_271227781_gt3nHI7H3CgUENSbM3V14x2QhPB04Glb.jpg"
+      style="float: right;"
+      width=70% 
+      margin:50px>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: space required after : in style

File: yogesh_khande/html/assignment3/Home.html [hunk 16]
```diff
@@ -44,69 +46,70 @@ <h1 style="color: rgb(78, 71, 175);"><center>Ambersoft</center></h1>
...
     <br>
     <br>
     <br>
-    <img src="https://t4.ftcdn.net/jpg/02/71/22/77/360_F_271227781_gt3nHI7H3CgUENSbM3V14x2QhPB04Glb.jpg"
-     style="float: left;" width="25%" margin:50px>
-     <p> contact us contact us contact us contact us contact us contact us contact us contact us contact us contact us
+       <img src="https://t4.ftcdn.net/jpg/02/71/22/77/360_F_271227781_gt3nHI7H3CgUENSbM3V14x2QhPB04Glb.jpg"
+        style="float: left;" 
+        width="25%" 
+        margin:50px>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: space required after : in style

File: yogesh_khande/html/assignment3/Home.html [hunk 17]
```diff
@@ -44,69 +46,70 @@ <h1 style="color: rgb(78, 71, 175);"><center>Ambersoft</center></h1>
...
+           <li>Sajan</li>
+           <li>Radha</li>
+        </ul>
 
     <br>
     <br>
     <br>
     <br>
-    </strong>
+       </strong>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: check indentation

File: yogesh_khande/html/assignment3/Home.html [hunk 18]
```diff
@@ -44,69 +46,70 @@ <h1 style="color: rgb(78, 71, 175);"><center>Ambersoft</center></h1>
...
-    </strong>
-    </p>
+      </strong>
+      </p>
     <br>
     <br>
     <br>
-    <center><h1 id="MI-11-MEAN">MI-11-MEAN</h1></center>
-    <center>
+       <center><h1 id="MI-11-MEAN">MI-11-MEAN</h1></center>
```
Comment (KomalShinde813) [inline yogesh_khande/html/assignment3/Home.html]: check indentation

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by KomalShinde813 — APPROVED** [review]

**Pull request merged**